### PR TITLE
fix (knora-ontologies): remove obsolete property

### PR DESCRIPTION
**Pull request by subotic**
### Summary

  - removing ``knora-base:systemProperty`` from the declaration of ``knora-base:institutionWebsite``

### Issues
  - #378

### CODE REVIEW

**Comment by subotic** [comment]
It is already there :) I also checked, that  every property has this.

File: knora-ontologies/knora-admin.ttl [hunk 1]
```diff
@@ -244,8 +244,6 @@
```
Comment (subotic) [inline knora-ontologies/knora-admin.ttl]: ``:objectCannotBeMarkedAsDeleted`` is already present.

**Comment by benjamingeer** [comment]
OK please merge!

**Comment by subotic** [comment]
Thanks :)

**Pull request merged**